src/data/sym/basic.lean

```diff
@@ -55,6 +57,20 @@ namespace sym
 
 variables {α : Type u} {n : ℕ} {s : sym α n} {a b : α}
 
+lemma coe_injective : injective (coe : sym α n → multiset α) := subtype.coe_injective
+
+@[simp, norm_cast] lemma coe_inj {s₁ s₂ : sym α n} : (s₁ : multiset α) = s₂ ↔ s₁ = s₂ :=
+coe_injective.eq_iff
+
+/--
+Construct an element of the `n`th symmetric power from a multiset of cardinality `n`.
+-/
+@[simps, pattern]
+abbreviation mk (m : multiset α) (h : m.card = n) : sym α n := ⟨m, h⟩
+
+lemma exists_mk : Π (s : sym α n), ∃ m h, s = mk m h
+| (mk m h) := ⟨m, h, rfl⟩
```

Comment:
I'm not sure I see the point in this lemma; `rcases exists_mk s with ⟨m, h, rfl⟩` is just a weird way to spell `cases s`.

Reply:
It's also not used in the the other PR.